QualificationsFramework/Examples/ACQF.json

```diff
@@ -0,0 +1,184 @@
+			"ceasn:hasTopConcept": [
+				"res:facetKnowledge",
+				"res:facetSkills",
+				"res:facetAutonomy"
+			]
+		},
+		{
+			"@id": "res:facetKnowledge",
+			"@type": "skos:Concept",
+			"skos:prefLabel": { "en": "Knowledge" },
+			"skos:definition": { "en": "Includes various kinds of knowledge such as facts, principles, and theories in various areas. The learning outcomes at all ten ACQF levels relate to formal, non-formal and informal learning. The knowledge domain comprises two sub-domains or elements:\n- Type of knowledge\n- Scope of knowledge" },
+			"skos:note": { "en": "In the context of ACQF \"Knowledge\" includes various kinds of knowledge such as facts, principles, and theories in various areas" }
+		},
+		{
+			"@id": "res:facetSkills",
+			"@type": "skos:Concept",
+			"skos:prefLabel": { "en": "Skills" },
+			"skos:definition": { "en": "Refer to the ability to use knowledge to respond to information and address problems/ issues. Skills include cognitive, communication, digital, green, innovation, practical and social skills. The learning outcomes at all ten ACQF levels relate to formal, non-formal and informal learning. The skills domain comprises three sub-domains:\n- Type of skills required\n- How the skills are used to respond to information\n- How skills are used to address types of problems" },
+			"skos:note": { "en": "In the context of ACQF \"Skills\" refer to the ability to use knowledge to respond to information and address problems. Skills include cognitive, communication, digital, green, innovation, practical and social skills." }
+		},
+		{
+			"@id": "res:facetAutonomy",
+			"@type": "skos:Concept",
+			"skos:prefLabel": { "en": "Autonomy and Responsibility" },
+			"skos:definition": { "en": "Refers to the context and extent of the application of autonomy and responsibility. The learning outcomes at all ten ACQF levels relate to formal, non-formal and informal learning. The autonomy and responsibility domain comprises 3 sub-domains:\n- the context in which autonomy and responsibility is applied\n- the extent to which autonomy is applied\n- the extent to which responsibility is applied.\nResponsibility as a domain of level descriptors has three elements: self, group outcomes, and resources." },
+			"skos:note": { "en": "In the context of ACQF \"Autonomy and responsibility\" refers to the context and extent of the application of autonomy and responsibility" }
+		},
+		{
+			"@id": "res:acqfLevels",
+			"@type": "asn:ProgressionModel",
+			"ceasn:name": { "en": "ACQF Level Descriptors" },
+			"ceasn:definition": { "en": "Each of the ten ACQF levels represents learning outcomes related to formal, non-formal and informal learning at that level. The ACQF level descriptors capture how the three domains of knowledge, skills and autonomy and responsibility (A&R) increase in breadth, depth and complexity when moving from lower to higher levels and are supported by a glossary explaining the terms." },
```

Comment:
`from lower to higher levels. They are supported by...`

Reply:
Not ours to fix.